parsel/selector.py

```diff
@@ -175,8 +175,11 @@ def xpath(self, query):
             return self.selectorlist_cls([])
 
         try:
-            result = xpathev(query, namespaces=self.namespaces,
-                             smart_strings=self._lxml_smart_strings)
+            nsp = dict(self.namespaces)
+            nsp.update(kwargs.pop('namespaces', {}))
```

Comment:
hm, @dangra , is second point to prevent a clash if users use `$namespaces` in their expressions and send a `namespaces="something"` of some sort?
I don't think  `lxml`'s `.xpath()` would like that anyway.